src/mongo_odm/nvp.hpp

```diff
@@ -160,13 +94,36 @@ class Nvp : public NvpCRTP<Nvp<Base, T>, T> {
     }
 
     /**
-     * Assignment operator that creaptes an update expression to set the field to a given value.
+     * Creates an update expression that sets the field to the given value.
      */
     constexpr UpdateExpr<Nvp<Base, T>, no_opt_type> operator=(const no_opt_type& val) const {
         return {*this, val, "$set"};
     }
 
-    virtual std::string get_name() const {
+    /**
+    * Creates an expression that sets a date value to the current date.
+    * This is only enabled for std::chrono::time/duration values, as well as b_date.
+    */
+    template <typename U = no_opt_type>
+    constexpr typename std::enable_if<is_date<U>::value, CurrentDateExpr<Nvp<Base, T>>>::type
+    operator=(const current_date_t& cd) const {
```

Comment:
you can just not name the parameter if you don't want to use it.

I.e.

operator=(const current_date_t&) const { return {*this, true}; }